R/api.R

```diff
@@ -5,8 +5,10 @@ api_build <- function(queue) {
   api$handle(endpoint_model_submit(queue))
   api$handle(endpoint_model_status(queue))
   api$handle(endpoint_plotting_metadata())
-  api$handle(endpoint_download_spectrum(queue))
-  api$handle(endpoint_download_summary(queue))
+  api$handle(endpoint_download_spectrum(queue, "GET"))
```

Comment:
I don't love this - is it worth separating out the endpoints here more so that the HEAD request is conformant, and shrink the shared code? This looks very different to the other endpoints this way

Reply:
I've added separate `endpoint_` function calls which have custom `returning` functions which ensure that no body is returned. Still goes through the same code path as the `GET` requests to ensure the same headers are applied.